dcos/http.py

```diff
@@ -97,7 +97,7 @@ def _request(method,
     except requests.exceptions.SSLError as e:
         logger.exception("HTTP SSL Error")
         msg = ("An SSL error occurred. To configure your SSL settings, "
-               "please run: `dcos config set core.ssl_verify <value>`")
```

Comment:
This is not exactly correct. ssl_verify can be True|False or path to CA_BUNDLE. It is the same as `verify` here: http://docs.python-requests.org/en/master/user/advanced/?highlight=ssl#ssl-cert-verification